src/components/signup/SignUpSelector.tsx

```diff
@@ -0,0 +1,56 @@
+import styled from '@emotion/styled';
+import { Box } from '@mui/material';
+import { ChangeEventHandler } from 'react';
+
+interface Props {
+  name: string;
+  errorMsg?: string;
+  onChange?: ChangeEventHandler<HTMLSelectElement>;
+}
+
+const SignUpSelector = ({ name, errorMsg, onChange }: Props) => {
+  const getFullYear = () => {
+    return Array(new Date().getFullYear() - 1980)
+      .fill(0)
+      .map((_, i) => new Date().getFullYear() - i);
+  };
+
+  return (
+    <Box
+      sx={{
+        width: '100%',
+        alignItems: 'center',
+        position: 'relative',
+        paddingBottom: '15px',
+      }}
+    >
+      <Selector name={name} onChange={onChange}>
+        <option value=''>출생연도</option>
+        {getFullYear().map((item, i) => (
+          <option key={i}>{item}</option>
+        ))}
+      </Selector>
+      {errorMsg ? <ErrorText>{errorMsg}</ErrorText> : null}
```

Comment:
```suggestion
      {errorMsg && <ErrorText>{errorMsg}</ErrorText>}
```

삼항 연산자 대신 and 연산자는 어떨까요?

Reply:
확인했습니다